index/constraints.txt

```diff
@@ -151,11 +154,11 @@ numpy==1.23.2
     #   rasterio
     #   snuggs
     #   xarray
-odc-apps-cloud==0.2.2.dev3382
+odc-apps-cloud==0.2.1
```

Comment:
has this not been updated?

Reply:
@pindge 0.2.1 is latest non dev version I believe
https://pypi.org/project/odc-apps-cloud/#history

Reply:
no, those are not build from pypi, they are built from https://packages.dea.ga.gov.au/odc-apps-cloud/

Reply:
Strange. The extra package repo was included (the "To update this," line in the new `constraints.txt` would be different if it wasn't). 

pip-compile doesn't include pre-releases of libraries by default, so maybe it was run last time with `--pre`?

Reply:
it depends on which code base you need, lot of the time, the code we need hasn't been released, the thing is, we need to learn how to do a release from odc-tools first.

Reply:
That makes sense.

In that case, we probably want to make it explicit by adding a min version in `requirements.txt` @JM-GA.

Change the odc-apps-cloud line to `odc-apps-cloud>=0.2.2.dev3382`, and then recompile.